docs: added more information to help endusers with proxies and ACL (#11543)
hashicorp/consul: website/content/docs/connect/proxies/integrate.mdx

the environment variables will be read and the client configured for you
automatically.

Alternatively, you may also use the flags `-token` or `-token-file` to provide the Consul ACL token.

<CodeTabs heading="Providing a Consul ACL Token" tabs={[ "Envoy", "Proxy" ]}>
<CodeBlockConfig language="shell-session">

```shell
 consul connect envoy -sidecar-for "web" -token-file=/etc/consul.d/consul.token
```

</CodeBlockConfig>

<CodeBlockConfig >

```shell
 $ consul connect proxy -sidecar-for "web" -token-file=/etc/consul.d/consul.token
```

</CodeBlockConfig>
</CodeTabs>

If TLS is enabled on Consul, you will also need to add the following environment variables _prior_ to starting the proxy:

- [`CONSUL_CACERT`](/commands#consul_cacert)
- [`CONSUL_CLIENT_CERT`](/commands#consul_client_cert)
- [`CONSUL_CLIENT_KEY`](/commands#consul_client_key)

> The `CONSUL_CACERT`, `CONSUL_CLIENT_CERT` and `CONSUL_CLIENT_KEY` can also be provided as CLI flags. Please see the [Consul connect documentation](/commands/connect/proxy) page for more details.

Reviewer: ```suggestion
The `CONSUL_CACERT`, `CONSUL_CLIENT_CERT` and `CONSUL_CLIENT_KEY` can also be provided as CLI flags. Refer to the [`consul connect proxy` documentation](/commands/connect/proxy) for details.
```
Just to make it a little leaner.
